http-api/src/lib.rs

```diff
@@ -232,6 +245,34 @@ pub async fn run(options: Options) -> anyhow::Result<()> {
     // Cleanup sessions
     tokio::spawn(cleanup_sessions_job(ctx.clone(), CLEANUP_SESSIONS_INTERVAL));
 
+    let app = Router::new()
+        .route("/", get(root_handler))
+        .layer(TraceLayer::new_for_http())
+        .layer(Extension(peer_id))
```

Comment:
Is there any reason why we're not simply passing context instead of a member?

Reply:
No real reason to pass data that won't be used, unless it's hard to pass individual members. Don't mind either way really, I think we pass context in a bunch of places as well.

Reply:
Performance is not an issue, it's the latter point you made, but consistency is maybe more valuable here :+1: